### PR TITLE
Fix project path issue

### PR DESCRIPTION
**Pull request by taquitos**
- Switch to relative path for project instead of hard-coded teammate's path
- update pod file to specify target and minimum deployment target
- update project and xcschemes files for Xcode 12

### CODE REVIEW

File: IntegrationTests/PurchaseTester/PurchaseTester.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:/Users/andresboedo/revenuecat/purchases-ios/IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj">
+      location = "group:PurchaseTester.xcodeproj">
```
Comment (aboedo) [inline IntegrationTests/PurchaseTester/PurchaseTester.xcworkspace/contents.xcworkspacedata:5]: oops 🤦

**Review by aboedo — APPROVED** [review]

**Review by vegaro — APPROVED** [review]

File: IntegrationTests/PurchaseTester/PurchaseTester.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (taquitos) [reply IntegrationTests/PurchaseTester/PurchaseTester.xcworkspace/contents.xcworkspacedata:5]: lol

File: IntegrationTests/PurchaseTester/Podfile [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
 end
+
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '12.0'
+    end
+  end
+end
```
Comment (aboedo) [inline IntegrationTests/PurchaseTester/Podfile:26]: I missed this when reviewing, why are we setting it here? also, shouldn't this be the same value as defined in line 3?

Comment (taquitos) [reply IntegrationTests/PurchaseTester/Podfile:26]: ooffff you're right, I missed this. I'll update it when I put my first PR up for PurchaseTest updates.

**Pull request merged**